index.html

```diff
@@ -8,8 +8,14 @@
     <title>My Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="mobile-version desktop-version">
       <nav class="hamburger-menu">
+        <ul class="menu-desktop-version">
```

Comment:
- [ ] I would kindly suggest you to work more on your mobile view to fit the figma design the hamburger menu need more adjustment according to the project requirement You need to make sure that your dekstop version looks like the one in the Figma design template that you chose.
- [ ] your mobile view 
![Screenshot_20230505-195238_1](https://user-images.githubusercontent.com/92458236/236547920-14fd3fd2-079f-4714-96ac-46cddc752fe3.jpg)